### PR TITLE
#210: Make private what is private

### LINKED ISSUE #210: Make private what is private
Now that the code has reached a mature stage, we should really make private those member functions and variables that are already private in practice (but not encoded as such).

In particular, all instance variables that have setters/getters should be made private.

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
### This PR:
- changed instance attributes to private
- fixed formatting
- fixed typos
- eliminated not used n_o in lbsRuntime
- added logging info of `Maximum number of objects transferred  at once` in transfer_stage of lbsRuntime
- made logging Hamming distance to optimal arrangementsdependent on `brute_force_optimization`

Closes #210 #215 #216

### CODE REVIEW

File: src/IO/lbsGridStreamer.py [hunk 1]
```diff
@@ -57,46 +57,46 @@ def __init__(self, points, lines, field_arrays, point_arrays, cell_arrays, lgr:
         """Class constructor
         """
         # Assign logger to instance variable
-        self.lgr = lgr
+        self.__lgr = lgr
```
Comment (ppebay) [inline src/IO/lbsGridStreamer.py:60]: `__logger`

File: src/IO/lbsVTDataReader.py [hunk 2]
```diff
@@ -73,18 +73,18 @@ class LoadReader:
...
-        self.file_prefix = file_prefix
+        self.__file_prefix = file_prefix
 
         # Data files(data loading) suffix
-        self.file_suffix = file_suffix
+        self.__file_suffix = file_suffix
 
         # Assign logger to instance variable
-        self.lgr = logger
+        self.__lgr = logger
```
Comment (ppebay) [inline src/IO/lbsVTDataReader.py:82]: `__logger`

File: src/IO/lbsVTDataWriter.py [hunk 3]
```diff
@@ -58,37 +58,37 @@ class VTDataWriter:
...
-    def __init__(self, phase: Phase, f="lbs_out", s="vom", output_dir=None, logger: Logger = None):
+    def __init__(self, phase: Phase, f: str = "lbs_out", s: str = "vom", output_dir=None, logger: Logger = None):
         """ Class constructor:
             phase: Phase instance
             f: file name stem
             s: suffix
         """
         # Assign logger to instance variable
-        self.lgr = logger
+        self.__lgr = logger
```
Comment (ppebay) [inline src/IO/lbsVTDataWriter.py:68]: `__logger`

File: src/IO/lbsWriterExodusII.py [hunk 4]
```diff
@@ -54,56 +54,56 @@ class WriterExodusII:
...
         """
 
         # Assign logger to instance variable
-        self.lgr = logger
+        self.__lgr = logger
 
         # Ensure that provided phase has correct type
         if not isinstance(p, Phase):
-            self.lgr.error("Could not write to ExodusII file by lack of a LBS phase")
+            self.__lgr.error("Could not write to ExodusII file by lack of a LBS phase")
```
Comment (ppebay) [inline src/IO/lbsWriterExodusII.py:72]: `__logger`

File: src/Applications/LBAF.py [hunk 5]
```diff
@@ -414,13 +414,8 @@ def get_output_file_stem(params, n_ranks):
...
 
     # Instantiate runtime
-    rt = Runtime(
-        phase,
-        params.work_model,
-        params.criterion,
-        params.order_strategy,
-        a_min_max,
-        lgr)
+    rt = Runtime(phase, params.work_model, params.criterion, params.order_strategy, a_min_max,
```
Comment (ppebay) [inline src/Applications/LBAF.py:417]: Please restore one parameter per line for legibility

**Review by ppebay — CHANGES_REQUESTED** [review]

File: src/Applications/LBAF.py (hunk 5 above)
Comment (marcinwrobel1986) [reply src/Applications/LBAF.py:417]: done

**Comment by ppebay** [comment]
Please one issue per PR next time

File: src/Model/lbsPhase.py [hunk 6]
```diff
@@ -59,47 +59,48 @@ class Phase:
...
         # Initialize empty list of ranks
-        self.ranks = []
+        self.__ranks = []
 
         # Default time-step/phase of this phase
-        self.phase_id = t
+        self.__phase_id = t
 
         # Initialize gossiping round
+        # TODO: seems not to be used anywhere
```
[the remark this answers is not in the document]
Comment (ppebay) [reply src/Model/lbsPhase.py:70]: Indeed `round_index` can be safely removed. Thanks.

[the remark this answers is not in the document]
Comment (ppebay) [reply src/Model/lbsPhase.py:70]: Regarding `self.dst_load` it *should be used*. Or at least it was at some point.
Can you please make an issue for that an assign it to me? I need to understand why and when it was removed.
Thanks for the catch @marcinwrobel1986

[the remark this answers is not in the document]
Comment (marcinwrobel1986) [reply src/Model/lbsPhase.py]: ok, done @ppebay

**Review by ppebay — APPROVED** [review]

**Pull request merged**